### ISSUE TITLE
centre word mark in header

### ISSUE THREAD

**Issue by Nasrine** [Nasrine/nasrinarts#3]
How to centre my word mark in header? @thomasjbradley

**Comment by thomasjbradley** [comment]
The code in this repository is the old version of your website. Have you committed & pushed recently?

**Comment by Nasrine** [comment]
yesterday  I copy and paste my new version "new" into the old version "nasrinearts". even the web tools.

**Comment by thomasjbradley** [comment]
You pushed it to GitHub?

**Comment by Nasrine** [comment]
You are right! I don't know why it is not updated even I committed on github desktop 2 hrs ago.

**Comment by Nasrine** [comment]
I just went to my computer/ web dev/ week4/ "nasrinarts" file,  and copy and paste the new one "new" over the old one

**Comment by Nasrine** [comment]
because my nasrinarts has the "README".md

**Comment by thomasjbradley** [comment]
Were you able to commit the files in the GitHub app?

**Comment by Nasrine** [comment]
Yes.
please look at your email for the screenshot.

**Comment by thomasjbradley** [comment]
Looks like it needs to be pushed still. Press the "Push" button.

**Comment by Nasrine** [comment]
Finally it updated!

**Comment by thomasjbradley** [comment]
I think it’s something to do with the grid units. Maybe remove the grid from the header and use `position` to get the logo into place?
